src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -657,6 +657,21 @@ public virtual int? SomeDacField{ get; set; }</value>
   <data name="PX1076Title" xml:space="preserve">
     <value>This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute which is not intended for public use</value>
   </data>
+  <data name="PX1077Fix" xml:space="preserve">
+    <value>Change the accessibility modifier to {0}</value>
```

Comment:
@EAndrosova please review diagnostic and code fix messages for PX1077 diagnostic